internal/aws/ssm_client.go

```diff
@@ -22,18 +23,26 @@ func GetSSMDocumentName() (string, error) {
 	return name, nil
 }
 
-// SSMAPI interface for SSM operations
-type SSMAPI interface {
+// SSMClientInterface defines the interface for SSM operations we need
+type SSMClientInterface interface {
```

Comment:
we also need count active session API. We need to enforce maximum active session number for each managed instance. Currently jorus is using 10.